### PR TITLE
Cleanup: remove redundant boolean check, use consistent error messages

### PR DESCRIPTION
**Pull request by hzoo**
From https://github.com/jscs-dev/node-jscs/pull/951#discussion_r26627322

Maybe the error message changes aren't as necessary?

### CODE REVIEW

**Comment by zxqfox** [comment]
Greatly done!

:+1: for merging.

File: lib/rules/require-trailing-comma.js [hunk 1]
```diff
@@ -60,17 +60,10 @@ module.exports.prototype = {
                 );
                 this._ignoreSingleValue = true;
             }
-            if ('ignoreSingleLine' in options) {
```
Comment (hzoo) [inline lib/rules/require-trailing-comma.js]: Only other change is removing this duplicate assert.

File: lib/rules/disallow-trailing-whitespace.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ module.exports.prototype = {
     configure: function(options) {
         assert(
             options === true || options === 'ignoreEmptyLines',
-            this.getOptionName() + ' option requires true value or "ignoreEmptyLines" string'
+            this.getOptionName() + ' option requires a value of value or "ignoreEmptyLines"'
```
Comment (mikesherov) [inline lib/rules/disallow-trailing-whitespace.js]: typo.

Comment (hzoo) [inline lib/rules/disallow-trailing-whitespace.js]: thanks

**Comment by mrjoelkemp** [comment]
:+1: @hzoo

**Pull request closed without merging**